src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/models/CCJContent.java

```diff
@@ -26,20 +26,22 @@ public CCJContent(Claim claim) {
         requireNonNull(claim);
 
         this.claimReferenceNumber = claim.getReferenceNumber();
-        TheirDetails defendant = claim.getClaimData().getDefendant();
+        CountyCourtJudgment countyCourtJudgment = claim.getCountyCourtJudgment();
+        TheirDetails defendant = countyCourtJudgment.getDefendant();
         this.defendantName = defendant.getName();
         this.defendantAddress = defendant.getAddress();
-        this.paymentRepaymentOption = claim.getCountyCourtJudgment().getPaymentOption().name();
+        this.paymentRepaymentOption = countyCourtJudgment.getPaymentOption().name();
         this.defendantEmail = defendant.getEmail().orElse(null);
-        this.amountToPayByDefendant = formatMoney(
-            ((AmountBreakDown) claim.getClaimData().getAmount()).getTotalAmount()
+        countyCourtJudgment.getRepaymentPlan().ifPresent(
```

Comment:
what about if the claimant asks for it to be paid by a set date or immediately, not with a repayment plan does this still work?

Reply:
you are right, problem, do I need to re calculate the remaining amount?

Reply:
I assume it would be: 
```
claim amount + interest till ccjCreatedAt  - any amount paid
```